views/phone_form.pug

```diff
@@ -0,0 +1,12 @@
+extends layout
+
+block content
+  h1 2-way Masked Phone Number SMS Conversation
+  form(method='post')
+    .form-group
+      label(for='phone-number') Phone Number
+      input#phone-number.form-control(name='phoneNumber' required='required')
+      label(for='expiration-time') Expiration Time
+      input#expiration-time.form-control(name='expirationTime' required='required')
+      small.form-text.text-muted Max time allowed to be inactive in minutes before been removed from the conversatiosn
```

Comment:
There's a typo here. It's `conversations` not `conversatiosn`

Reply:
👍